### ISSUE TITLE
Missing string in translations

### ISSUE THREAD

**Issue by hide-me** [bludit/bludit#1188]
### Describe your problem
**Missing string (untranslatable)**

Strings in CMS core and strings it lang files not same:
file: _bl-kernel/admin/views/edit-content.php_
`'tip'=>$L->g('Write the tags separated by comma'),`
file: _bl-kernel/admin/views/new-content.php_
`'tip'=>$L->g('Write the tags separated by comma')`

and in all lang files:
`"write-the-tags-separated-by-commas": "Write the tags separated by comma.",`

### Steps to reproduce the problem
1. Open project
2. Check described files

### Bludit version
3.12.X (current)

### PHP version
Doesn't matter

### PHP logs
Doesn't matter

**Comment by lightmat** [comment]
Well spotted.
This is because of the lack of "s" in comma
You should create a pull request so that the error can be corrected.

**Comment by hide-me** [comment]
> This is because of the lack of "s" in comma

You are quite right, unfortunately I haven't permission to make any changes (or pull requests) in Bludit core.

I hope anybody will fix it instead of me or will give access to do it for my account.

**Comment by dignajar** [comment]
@hide-me really you don't have permissions ? that is strange is open public the PR..

**Comment by dignajar** [comment]
The correct string would be `Write the tags separated by a commas` correct?

**Comment by lightmat** [comment]
> The correct string would be `Write the tags separated by a commas` correct?

Yes it is :) @dignajar

**Comment by dignajar** [comment]
Hello, with the new version of Bludit v4.0 rc1, I would like to close the old Github issues. If you feel that your issue is not resolved in the latest version, create a new ticket.

- Help and Support use the Forum https://forum.bludit.org
- Bugs and new requests here in Github https://github.com/bludit/bludit/issues
